py3status/modules/uname.py

```diff
@@ -26,7 +26,7 @@ class Py3status:
     """
     """
     # available configuration parameters
-    format = '{system} {release} {machine}'
+    format = '{system} {release}'
```

Comment:
@ultrabug what is you take on removing `{machine}`?  

@lasers Is there a strong reason you want it removed.  `{system}` also doesn't change but the fact that it is constant does not mean that it is not useful for many users.

Reply:
My subjective opinion is that I think most of the users would prefer not having `{machine}` in their statusbar because this placeholder does not change 99% of the time. Again, subjective... as we're keeping `{system}` as it does not change either... meaning we could remove `{system}` too... and have it only print `{release}`. I like having `Linux` prefix.

It's all subjective. Having it printing no more than `{release}` may be a good choice too. I'm just putting in what I think is good and hopefully people like adding `uname {}` more than adding `uname{} + config`. If you're not sure, then why don't we include `{node}` and `{processor}` too as they don't change much?

Also, I can say we can make it `None` as default (not biased), then let users add what they want, but I think that would suck terribly as this is a simple module and we shouldn't force users to add what they want... This exception is made on large modules (eg, `window_manipulation`).

So maybe we should use `{release}` here as it's the only thing that can change. The next best choice is this PR. Having `Linux` prefix to make it more smoothing. Anyhow, that's my reason.